src/libraries/System.Private.CoreLib/src/System/Reflection/Emit/SymbolType.cs

```diff
@@ -14,8 +15,25 @@ internal enum TypeKind
     }
 
     // This is a kind of Type object that will represent the compound expression of a parameter type or field type.
+#if MONO
+    [StructLayout(LayoutKind.Sequential)]
+#endif
     internal sealed class SymbolType : TypeInfo
     {
+        #region Data Members
```

Comment:
Fields that are read by the runtime are usually placed into the .Mono.cs partial class.

Reply:
Yeah, but the fields are used for both Mono and CLR, not that sure what you are suggesting.

Reply:
Moved `#region Sync with MonoReflectionDerivedType in object-internals.h` into .Mono.cs as comment for the type